wman-backend/Wman.Repository/Classes/WorkEventRepo.cs

```diff
@@ -32,14 +35,14 @@ public async Task Delete(int element)
 
         public IQueryable<WorkEvent> GetAll()
         {
-            return this.db.WorkEvent.AsNoTracking().Include(x => x.Address).Include(x => x.AssignedUsers);
+            return this.db.WorkEvent.AsNoTracking().Include(x => x.Address).Include(x => x.AssignedUsers).ThenInclude(x => x.ProfilePicture).Include(x => x.Labels);
```

Comment:
As you would like to return every `WorkEvent` ordered by its `EstimatedStartDate` at `CalendarEventLogic` I would suggest the following solution instead of appending `.OrderBy(x => x.EstimatedStartDate)` to every query in `CalendarEventLogic`.
```suggestion
            return this.db.WorkEvent.AsNoTracking().Include(x => x.Address).Include(x => x.AssignedUsers).ThenInclude(x => x.ProfilePicture).Include(x => x.Labels).OrderBy(x => x.EstimatedStartDate);
```